core/base-service/openapi.js

```diff
@@ -241,7 +241,9 @@ function category2openapi(category, services) {
           name: 'style',
           in: 'query',
           required: false,
-          description: 'If not specified, the default style is "flat".',
+          description: `If not specified, the default style is "${
```

Comment:
Maybe:
```suggestion
          description: `If not specified, the default style for this badge is "${
```

To make sure that users don't assume the default for this common parameter is the same for all badges?